editor/src/components/canvas/commands/adjust-css-length-command.ts

```diff
@@ -70,237 +58,159 @@ export function adjustCssLengthProperties(
   }
 }
 
-interface UpdatedPropsAndCommandDescription {
-  updatedProps: JSXAttributes
-  commandDescription: string
-}
-
-export const runAdjustCssLengthProperties: CommandFunction<AdjustCssLengthProperties> = (
+export const runAdjustCssLengthProperties = (
```

Comment:
`runAdjustCssLengthProperties` is refactored not to read directly from the inline style prop